zeplin/AppDelegate.swift

```diff
@@ -43,6 +44,9 @@ extension AppDelegate {
     func application(_ application: UIApplication,
                      didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         window = splashWindow
+        
+        let onesignalSettings = [kOSSettingsKeyAutoPrompt: false]
+        OneSignal.initWithLaunchOptions(onesignalSettings, appId: "3b3075a1-b7b0-4f9a-a72c-2856fa865ca4")
```

Comment:
App Id'yi daha güzel saklama yöntemi var mı? BE'den gelse mesela?

Reply:
Surekli degisen bir sey degil